packages/testing/src/index.ts

```diff
@@ -396,33 +401,31 @@ export const defaultActivityInfo: activity.Info = {
 export class MockActivityEnvironment extends events.EventEmitter {
   public cancel: (reason?: any) => void = () => undefined;
```

Comment:
Can we document that the `context` is shared for any activity that is run in this instance and that once `cancel` is called, any further activity runs in this env will be automatically cancelled?